docs/source/torch_xla.rst

```diff
@@ -0,0 +1,92 @@
+torch_xla
+===================================
+
+PyTorch runs on TPUs with the `torch_xla`_ package.
+
+XLA devices
+----------------------------------
+
+The XLA device type defines virtual 'xla' devices backed by real hardware.
+'xla:0' is your machine's CPU and is useful for testing.
```

Comment:
I would not expose that device is a given ordinal, as it might be subject to change.
Documenting properly `xm.xla_device()` and ask users to use that should suffice.